src/components/Modal/AletheiaModal.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, Modal } from "antd";
+import styled from "styled-components";
+import colors from "../../styles/colors";
+
+export const AletheiaModal = styled(Modal)`
```

Comment:
Any reason for creating a new file instead of using the AletheiaModal.style already created?